docs/runbooks/KubeVirtVMGuestMemoryHigh.md

```diff
@@ -0,0 +1,225 @@
+# KubeVirtVMGuestMemoryHigh
+
+## Meaning
+
+This alert fires when a Virtual Machine has been consuming more than 85% of
+its available memory for more than 5 minutes, as seen from the guest OS
+perspective.
+
+## Impact
+
+High memory usage in a VM can lead to:
+- **Performance degradation** - Applications may become slow due to memory pressure
+- **Memory swapping** - Guest OS starts using swap space, severely impacting performance
+- **Application crashes** - Out-of-memory (OOM) conditions may kill processes
+- **System instability** - Critical system processes may fail due to memory exhaustion
+- **Cascading failures** - Memory pressure can affect the entire application stack
+
+## Diagnosis
+
+### 1. Identify the Affected VM
+
+The alert provides the following information: VM Name, Namespace, Node and
+Memory Usage.
+
+### 2. Check VM Memory Allocation
+
+```bash
+# Get VM configuration
+kubectl get vm <vm-name> -n <namespace> -o yaml
+
+# Check current VMI status
+kubectl get vmi <vm-name> -n <namespace> -o yaml
+```
+
+Look for:
+- `spec.domain.memory.guest` - Guest memory allocation
+- `spec.domain.resources.requests.memory` - Memory resource requests
+- `spec.domain.resources.limits.memory` - Memory resource limits
```

Comment:
Are these values really relevant?

Reply:
Check how much memory is assigned to the VM and check from within the VM if it is actually able to make use of it.